src/main/java/cubicchunks/lighting/LightingManager.java

```diff
@@ -118,9 +118,19 @@ private boolean updateDiffuseLight(Cube cube, int localX, int localZ, int minY,
         }
         int blockX = localToBlock(cube.getX(), localX);
         int blockZ = localToBlock(cube.getZ(), localZ);
+        
+        CubePos minLoad = cube.getCoords().add(-1, -1, -1);
+        CubePos maxLoad = cube.getCoords().add(1, 1, 1);
+        
+        if (!world.testForCubes(minLoad, maxLoad,
+                c -> c != null && !(c instanceof BlankCube))) {
+            return false;
+        }
 
+        ILightBlockAccess blocks = new FastCubeBlockAccess(world.getCubeCache(), cube, 1);
```

Comment:
It accidentally works, 1 cube radius isn't enough because of reasons explained in previous comment.